lib/i18n/strings_cs.i18n.json

```diff
@@ -0,0 +1,156 @@
+      "system": "Systémový",
+      "light": "Světlý",
+      "dark": "Tmavý"
+    },
+    "accentColorPicker": {
+      "pickAColor": "Vyberte barvu",
+      "confirm": "Potvrdit"
+    },
+    "straightenDelay": {
+      "off": "Vypnuto",
+      "regular": "Standardně",
+      "slow": "Dlouze"
+    }
+  },
+  "login": {
+    "title": "Přihlásit se",
+    "feedbacks": {
+      "checkUsername": "Zkontrolujte prosím své uživatelské jméno a heslo.",
+      "enterNcPassword": "Zadejte prosím své heslo k NextCloud účtu.",
+      "enterEncPassword": "Zadejte prosím své heslo pro šifrování.",
+      "checkUrl": "Zadejte prosím správnou URL adresu.",
+      "ncLoginFailed": "Přihlašování selhalo, zkontrolujte prosím zadané údaje a připojení k internetu.",
+      "encLoginFailed": "Přihlašování selhalo, zkontrolujte prosím své heslo pro šifrování.",
+      "loginSuccess": "Přihlášení bylo úspěšné! Vyčkejte prosím, nastavíme vše potřebné..."
+    },
+    "form": {
+      "useCustomServer": "Chci využít vlastní NextCloud server",
+      "customServerUrl": "URL adresa vlastního NextCloud serveru",
+      "username": "Přihlašovací jméno nebo e-mail",
+      "ncPassword": "Heslo k NextCloud účtu",
+      "encPassword": "Heslo pro šifrování",
+      "agreeToPrivacyPolicy(rich)": "Přihlášením souhlasíte se ${linkToPrivacyPolicy(Zásadami ochrany osobních údajů)}.",
+      "login": "Přihlásit se"
+    },
+    "signup(rich)": "Ještě nemáte účet? ${linkToSignup(Zaregistrujte se)}!",
+    "status": {
+      "loggedIn": "Jste přihlášeni pomocí Nextcloud účtu",
+      "loggedOut": "Jste odhlášení",
+      "tapToLogin": "Klepněte pro přihlášení pomocí NextCloud účtu"
+    },
+    "faq": [{
+      "q": "Co je to NextCloud?",
+      "a": "Nextcloud je mimo jiné osobní server pro synchronizaci souborů. Můžete používat výchozí NextCloud server poskytovaný aplikací Saber, použít server hostovaný třetí stranou nebo vlastní server pro plnou kontrolu nad svými daty."
+    }, {
+      "q": "K čemu je heslo pro šifrování?",
+      "a": "Heslo pro šifrování je použito k zašifrování vašich dat před jejich odesláním na server. Není uloženo na serveru a je použito pouze aplikací Saber pro zašifrování a dešifrování dat.\n\nBez jeho znalosti nemůže nikdo přistoupit k vašim poznámkám uloženým na serveru. To také znamená, že pokud ho zapomenete, ztratíte přístup ke svým datům."
+    }, {
+      "q": "Proč používat dvě hesla?",
+      "a": "Heslo k NextCloud účtu se používá k přístupu do cloudu. Heslo pro šifrování \"utají\" vaše data ještě před tím, než se na cloud dostanou.\n\nI když by někdo získat přístup k vašemu NextCloud účtu, vaše poznámky zůstanou v bezpečí zašifrované vlastním heslem. To vám poskytuje druhou vrstvu ochrany vašich dat."
+    }]
+  },
+  "appInfo": {
+    "licenseNotice": "Saber  Copyright (C) 2022  Adil Hanney\nTento program je poskytován bez jakékoliv záruky. Jedná se o software poskytovaný zdarma, který je možné šířit při splnění daných podmínek.",
```

Comment:
```suggestion
    "licenseNotice": "Saber  Copyright (C) 2023  Adil Hanney\nTento program je poskytován bez jakékoliv záruky. Jedná se o software poskytovaný zdarma, který je možné šířit při splnění daných podmínek.",
```

Reply:
Good catch! I'll change this to 2022-yyyy (where yyyy is the year of the latest release) after this PR gets merged

Reply:
Oh, yes 😅. I just copied what I found for English version.

Reply:
Btw, could I change the date to interval now or you need it in another PR?

Reply:
No don't worry I'll handle that. It needs some minor changes to the code to make it automatically update

Reply:
Ok, so changes are ready to be merged.